tests/test_slt_filtering.py

```diff
@@ -33,7 +33,7 @@ def full_slt():
 
 def test_filter_fn_on_logic_tree_branches(full_slt):
     # functional style
-    assert len([filt_branch for filt_branch in filter(lambda obj: obj.fslt.short_name == 'HIK', full_slt)]) == 9
+    assert len([filt_branch for filt_branch in filter(lambda obj: obj.branch_set.short_name == 'HIK', full_slt)]) == 9
```

Comment:
@chrisdicaprio here's an example. where the **fslt** prop is superceded by  **branch_set**. Maybe we should make both work and add deprecation warnings??